### PR TITLE
feat: Remove limitation and add new metrics to list PUL-1566

### PR DESCRIPTION
**Pull request by claudiacarpinteiro**
With Bitbucket V3 delivery, Pulse will start to show the drill-down metrics:

- Changes & Reviews
- Work in progress

### :eyes: Live preview
https://feat-bitbucket-v3-reviews-wip-pul-1--docs-pulse.netlify.app/one-click-integrations/bitbucket-integration/
https://feat-bitbucket-v3-reviews-wip-pul-1--docs-pulse.netlify.app/metrics/accelerate-changes/
https://feat-bitbucket-v3-reviews-wip-pul-1--docs-pulse.netlify.app/metrics/accelerate-wip/
https://feat-bitbucket-v3-reviews-wip-pul-1--docs-pulse.netlify.app/one-click-integrations/github-integration/#see-also

### :construction: To do
-   [x] Update content to reflect the new metrics or Bitbucket integration
-   [x] Request validation
-   [x] Wait for feature release

### CODE REVIEW

File: docs/one-click-integrations/bitbucket-integration.md [hunk 1]
```diff
@@ -83,6 +86,13 @@ The table below lists the data that the Bitbucket integration collects from your
     </td>
     <td>Deployment frequency and Change failure rate on the <a href="../../metrics/accelerate/">Accelerate Overview dashboard</a></td>
 </tr>
+<tr>
+    <td>Pull requests</td>
+    <td>
+        -
+    </td>
+    <td><a href="../../metrics/accelerate-changes/">Changes & Reviews dashboard</a>,<br/><a href="../../metrics/accelerate-wip/">Work in Progress dashboard</a></td>
+</tr>
```
Comment (claudiacarpinteiro) [inline docs/one-click-integrations/bitbucket-integration.md:95]: @afsmeira I'm updating the collected data table to include the new metrics for BB.
I'm adding this line to the table as it is in the GitHub integration, but it's missing the details about the mapping to Pulse data model.
Do we have any additional mapping for this line? Or does it make sense to merge this line with the previous one? (does the same mapping apply?)

File: docs/metrics/accelerate-wip.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # Work in progress metrics
 
-Pulse calculates the work in progress metrics based on the status of all pull requests targeting any base branch in your GitHub repositories.
+Pulse calculates the work in progress metrics based on the status of all pull requests targeting any base branch in your repositories.
 
 !!! important
-    You must install the [GitHub integration](../one-click-integrations/github-integration.md) for Pulse to calculate the work in progress metrics.
+    You must install the [GitHub integration](../one-click-integrations/github-integration.md) or the [Bitbucket integration](../one-click-integrations/bitbucket-integration.md) for Pulse to calculate the work in progress metrics.
```
Comment (claudiacarpinteiro) [inline docs/metrics/accelerate-wip.md:6]: @mrfyda I updated this warning to consider the Bitbucket integration (it was only referring to GitHub), but I'm not sure why we have this information here... We don't have it in Changes & reviews metrics, for example. Should we add a similar warning there?

Also, is it really required to install the GitHub integration for Pulse to calculate the WIP metrics? Can't the customer choose to use the Pulse CLI or the API instead?

Comment (prcr) [reply docs/metrics/accelerate-wip.md:6]: > that's right. the API only ingest data for the Accelerate dashboard. data for the Changes and WIP dashboard is exclusive for the GH and BB integration; and data for the Lead/Cycle time is exclusive from the Jira integration

I confirm this was the reason that led me to mention that the GitHub integration was required for calculating the metrics. :ok_hand:

File: docs/one-click-integrations/bitbucket-integration.md (hunk 1 above)
Comment (afsmeira) [reply docs/one-click-integrations/bitbucket-integration.md:95]: We don't do any mapping to the pull request data coming from Bitbucket. We simply cherry pick what we need and store it on our side to use for those dashboards.

**Review by mrfyda — APPROVED** [review]

**Comment by mrfyda** [comment]
(I've asked Codacy Support was is going on with the status check. it's pending for 15 hours)

**Review by afsmeira — APPROVED** [review]

**Review by prcr — APPROVED** [review]

**Pull request merged**